src/lib/AnnotationControls.ts

```diff
@@ -167,7 +168,7 @@ export default class AnnotationControls {
             this.updateButton(mode);
         }
 
-        onClick({ event });
+        onClick({ event, mode: this.currentMode });
```

Comment:
Could we make `AnnotationControls` even dumber by removing `handleClick` and just calling `onClick` whenever one of the annotation buttons are clicked? Then give the responsibility to `BaseViewer` to get the new mode from the `AnnotationFSM` and call back in `this.annotationControls.setMode(...)`?

It would make the flow of data more consistent for `AnnotationControls`